.travis.yml

```diff
@@ -1,8 +1,11 @@
 ---
+sudo: required
+services:
+    - docker
 language: python
 python: 2.7
 cache: pip
-install:
-  - "pip install vim-vint==0.3.9"
-script:
-  - "vint -s ."
+install: |
+    pip install vim-vint==0.3.9
+script: |
+    vint -s . && make test
```

Comment:
If we put them line after line, we will only get the error code of the last line. I've implemented the solution I found and it's working well.